target/linux/ath79/dts/qca9531_compex_wpj531.dts

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "compex,wpj531", "qca,qca9531";
+	model = "Compex WPJ531 (16MB flash)";
```

Comment:
If there are different (official) flash size variants for this devices, consider renaming it to "compex,wpj531-16m".
If not, remove the information about flash size here and add it to the commit message.

Reply:
I haven't seen anything different, personally.
Compex offers changing the flash size for a batch (`Increase NOR Flash up to 32 M`, page 7 https://compex.com.sg/wp-content/uploads/2019/07/wpj531-v3.1-sl-172-1.pdf)
The description was taken over from ar71xx.

Reply:
Hmm, sounds to me like this one should be compex,wpj531-16m, if an alternate version is available from vendor (still if it's only on request).
Might be that other people see it differently, since OpenWrt does not support user modifications. But this is a vendor modification ...
If you rename, you will also have to rename DTS and device node in generic.mk

Reply:
i also think the 16m should be present in the suffix - although it's arguable nothing should break on a 32m board.

Reply:
Okay, then do it this way.
Need to change compatible, DTS name and device definition name in generic.mk.